### PR TITLE
Modernized ALP description

### PR DESCRIPTION
**Pull request by tbazant**
### Description

The old description was too technically oriented, this may sound better to people.

### CODE REVIEW

File: articles/ALP.asm.xml [hunk 1]
```diff
@@ -244,12 +244,10 @@
...
-                Running a minimal OS where software is managed in transactions
-                and applications are containerized has several advantages over
-                the traditional OS. For example, better isolation of the core
-                OS from the application layer. &productnameshort; is simply a
-                more stable and secure system powered by running &suse;-tuned
-                workloads.
+                &productnameshort; is a minimal operating system that allows
+                customers to run their workloads in a containerized or
+                virtualized form. Typically, &productnameshort; is deployed on
+                bare-metal servers and runs data center oriented workloads.
```
Comment (dariavladykina) [inline articles/ALP.asm.xml:250]:
```suggestion
                bare-metal servers and runs data center-oriented workloads.
```

File: concepts/alp.xml [hunk 2]
```diff
@@ -22,127 +22,49 @@
...
-    </para>
-  </section>
-  <section xml:id="how-it-works-alp">
-    <title>Core components of &productnameshort;</title>
-    <para>
-      &productname; consists of the following components:
+      &productnameshort; is a minimal operating system that allows customers to
+      run their workloads in a containerized or virtualized form. Typically,
+      &productnameshort; is deployed on bare-metal servers and runs data center
+      oriented workloads. &productnameshort; is best described by the following
```
Comment (dariavladykina) [inline concepts/alp.xml:28]:
```suggestion
      &productnameshort; is deployed on bare-metal servers and runs data
      center-oriented workloads. &productnameshort; is best described by the following
```

File: concepts/alp.xml [hunk 3]
```diff
@@ -22,127 +22,49 @@
...
+        <term>Minimalistic</term>
         <listitem>
           <para>
-            The core of &productnameshort; which runs all required services. It
-            is an immutable operating system with a read-only root file system.
-            The file system is modified by transactional updates which utilize
-            the snapshotting feature of Btrfs.
+            &productnameshort; provides the bare minimum to run workloads and
+            services as containers or virtual machines. Its default software
+            collection is carefully pre-selected to include only packages
```
Comment (dariavladykina) [inline concepts/alp.xml:38]:
```suggestion
            collection is carefully preselected to include only packages
```

**Review by dariavladykina — COMMENTED** [review]
LGTM! Just a couple nits. Thanks!

**Review by taroth21 — APPROVED** [review]
LGTM, many thanks!

**Pull request merged**